factcast-store/src/main/java/org/factcast/store/internal/query/CurrentStatementHolder.java

```diff
@@ -33,7 +33,13 @@ public void close() {
       if (statement != null) {
         log.info("Canceling statement " + statement);
         try {
+
           statement.cancel();
+
+          // we have to rollback the tx on the undderlying transaction
```

Comment:
double d

Reply:
Its really that important!!! :D

Reply:
just trying to help :man_shrugging:

Reply:
I meant the extra d underlines the importance 😅 adjusted the comment